pkg/token/options.go

```diff
@@ -8,6 +8,7 @@ const (
 	ServicePrincipalLogin = token.ServicePrincipalLogin
 	MSILogin              = token.MSILogin
 	WorkloadIdentityLogin = token.WorkloadIdentityLogin
+	AzurePipelinesLogin   = token.AzurePipelinesLogin
```

Comment:
please remove the changes in pkg/token/options.go and options_ctor.go as we do not plan to support it here.